sideboard/config.py

```diff
@@ -13,57 +13,69 @@ class ConfigurationError(RuntimeError):
     pass
 
 
-def os_path_split_asunder(path, debug=False):
+def get_dirnames(pyname, plugin):
```

Comment:
I believe I added enough docstring comments in this file to explain what's going on, but let me know if anything is confusing.